### ISSUE TITLE
favicon

### ISSUE THREAD

**Issue by klonuo** [rtfd/readthedocs.org#2324]
Please provide favicon in .ico format, which is to be expected for https://readthedocs.org/favicon.ico or content-type 'image/x-icon', instead PNG.
Or just use PNG image and provide right content-type:

```html
<link rel="icon" type="image/png" href="https://readthedocs.org/favicon.png" />
```

Thanks

---

Hm... you do provide right icon in HTML source code as:

```html
<link rel="icon" type="image/png" href="https://media.readthedocs.org/images/favicon.png">
```

but anyways https://readthedocs.org/favicon.ico exists as PNG image.

Offending file is at https://github.com/rtfd/readthedocs.org/blob/master/media/images/favicon.ico which has wrong extension and the same file exists also as https://github.com/rtfd/readthedocs.org/blob/master/media/images/favicon.png. 

Gimp can convert png to ico, or imagemagick, or else.

**Comment by tapasweni-pathak** [comment]
I would like to pick this up. As right icon is provided in HTML source code, and the related file exist at https://github.com/rtfd/readthedocs.org/blob/master/media/images/favicon.png, should  https://github.com/rtfd/readthedocs.org/blob/master/media/images/favicon.ico be removed to fix this?

What is the right way to fix this issue?

**Comment by berkerpeksag** [comment]
Hello @tapasweni-pathak, sorry, this has already been resolved in #2334. I'd suggest to look the list at https://github.com/rtfd/readthedocs.org/issues?q=is%3Aissue+is%3Aopen+label%3A%22Good+First+Bug%22 if you have time to work on other issues.

Thanks!

**Comment by tapasweni-pathak** [comment]
Cool. Sure.
